examples/cbtc/serverless/handler.ts

```diff
@@ -0,0 +1,241 @@
+import { BitcoinChainEvent, StacksChainEvent, StacksTransactionEventType, StacksFTBurnEventData } from "@hirosystems/orchestra-types";
+
+import {
+  getNonce,
+  makeContractCall,
+  broadcastTransaction,
+  AnchorMode,
+  PostConditionMode,
+  addressFromHashMode,
+  AddressHashMode,
+  TransactionVersion,
+  standardPrincipalCVFromAddress,
+  uintCV,
+  addressToString,
+} from '@stacks/transactions';
+import { StacksTestnet } from '@stacks/network';
+import { principalCV } from "@stacks/transactions/dist/clarity/types/principalCV";
+const Script = require('bitcore-lib/lib/script');
+const Opcode = require('bitcore-lib/lib/opcode');
+const Networks = require('bitcore-lib/lib/networks');
+const Transaction = require('bitcore-lib/lib/transaction');
+const PrivateKey = require('bitcore-lib/lib/privatekey');
+const Signature = require('bitcore-lib/lib/crypto/signature');
+const { Output, Input } = require('bitcore-lib/lib/transaction');
+
+interface HttpEvent {
+  routeKey: string,
+  body: string
+  authorization: string,
+}
+
+const cbtcAuthority = {
+  secretKey: "7287ba251d44a4d3fd9276c88ce34c5c52a038955511cccaf77e61068649c17801",
```

Comment:
Is there an easy way to do this via environment variables or "secrets"or something, so that we're not demoing unsafe behavior that we don't want users to copy?

Or if there's not a simple way to do that for the example, let's at least add a comment here saying that it is unsafe and not recommended.

Reply:
Good idea, I'll use an env var.